checks/default.list

```diff
@@ -1,5 +1,5 @@
 # RFC 1918 Checks (Approximation)
-(127\.([0-9\.]{1,3}){3}|10\.([0-9\.]{1,3}){3}|172\.[123][0-9]\.([0-9\.]{1,3}){2}|192\.168\.([0-9\.]{1,3}){2})
+127\.(([0-9]){1,3}\.){2}([0-9]){1,3}|10\.(([0-9]){1,3}\.){2}[0-9]{1,3}|172\.[123][0-9]\.([0-9\.]{1,3}){2}|192\.168\.([0-9]{1,3}\.[0-9]{1,3})
```

Comment:
Good catch! I think this is closer, but it doesn't fix 172.* addresses. I started with your suggestion and paired down unnecessary capture groups and fixed the 172 address search. Thoughts? https://regex101.com/r/LBWFC8/1

Reply:
Also, all the expressions in the list need to be enclosed by parenthesis for the capture group matching. Thank you!